open_rarity/resolver/opensea_api_helpers.py

```diff
@@ -163,8 +201,11 @@ def opensea_traits_to_token_metadata(asset_traits: list) -> TokenMetadata:
     )
 
 
-def get_tokens_from_opensea(
-    opensea_slug: str, token_ids: list[int]
+async def get_tokens_from_opensea(
```

Comment:
probably dont need both functions anymore - i would just merge the two and rename

Reply:
👍  was planning on that.